gdk-pixbuf/src/auto/pixbuf_format.rs

```diff
@@ -9,7 +9,7 @@ glib::wrapper! {
     pub struct PixbufFormat(Boxed<ffi::GdkPixbufFormat>);
 
     match fn {
-        copy => |ptr| ffi::gdk_pixbuf_format_copy(mut_override(ptr)),
```

Comment:
This is not due to my patch... not sure if this is correct or if current gir master is wrong

Reply:
Yes that's correct. There were more gir changes since the last time things were regenerated